src/index.css

```
@@ -93,589 +90,558 @@
+     Border strategy: inset box-shadow rim lighting instead of
+     a flat border — creates a directional highlight that reads
+     as physical glass thickness catching light from top-left.
+     Three tiers of visual weight: card > panel > input/subtle
+     ============================================================ */
+
+  /* Cards — most prominent surface, heaviest rim */
   .glass-card {
-    @apply rounded-3xl border border-white/10 bg-white/5 backdrop-blur-3xl saturate-150 shadow-glass transition-all duration-300;
-    @apply dark:border-white/10 dark:bg-white/5;
+    @apply rounded-3xl bg-white/5 backdrop-blur-3xl saturate-150 transition-all duration-300;
+    border: none;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)